testing/Cargo.toml

```diff
@@ -9,15 +9,17 @@ default = []
 nightly = ["rocket", "rocket_codegen", "askama/with-rocket"]
 
 [dependencies]
-askama = { path = "../askama", version = "*", features = ["with-iron", "serde-json"] }
+askama = { path = "../askama", version = "*", features = ["iron", "serde-json"] }
```

Comment:
I think this change is probably responsible for the test failures, the feature in `askama` is still called `with-iron` here, right?

Reply:
Also, I'd like to have the Cargo manifest changes for the `testing` crate in the earlier `test` commit. I can take care of that if you prefer, though.